settings/installer/ospd/images/build.yml

```diff
@@ -0,0 +1,14 @@
+---
+installer:
+    images:
+        task: build
+        overcloud:
+            guest_image: "http://download.eng.bos.redhat.com/brewroot/packages/rhel-guest-image/7.2/20160302.0/images/rhel-guest-image-7.2-20160302.0.x86_64.qcow2"
```

Comment:
Ok, I'll solve this with new private settings as we don't have image link defined anywhere.